<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubtitleItem</class>
 <widget class="QWidget" name="SubtitleItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Subtitle item</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
     <item>
      <widget class="QCheckBox" name="isSelected">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="orderLbl">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="subtitleItemSettings">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="srt_editor_resources.qrc">
         <normaloff>:/subtitle/assets/setting.png</normaloff>:/subtitle/assets/setting.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0,1">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QSpinBox" name="startHourLbl"/>
         </item>
         <item>
          <widget class="QSpinBox" name="startMinuteLbl"/>
         </item>
         <item>
          <widget class="QSpinBox" name="startSecondLbl"/>
         </item>
         <item>
          <widget class="QSpinBox" name="startMiliSecondLbl">
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>==&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSpinBox" name="endHourLbl"/>
         </item>
         <item>
          <widget class="QSpinBox" name="endMinuteLbl"/>
         </item>
         <item>
          <widget class="QSpinBox" name="endSecondLbl"/>
         </item>
         <item>
          <widget class="QSpinBox" name="endMiliSecondLbl">
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="subtitleContent"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="srt_editor_resources.qrc"/>
 </resources>
 <connections/>
</ui>
